src/__test__/Crypto.test.js

```diff
@@ -0,0 +1,18 @@
+import renderer from 'react-test-renderer';
```

Comment:
- [ ] Well done with the component testing so far. However, you're still missing the following:
1- Please test more components
2- Please write integration tests using react testing library
    You can use mocking data fetching to test your redux pure functions.

Reply:
Thank you for the review